Duplicati/Library/Modules/Builtin/SendMail.cs

```diff
@@ -231,42 +231,10 @@ protected override void SendMessage(string subject, string body)
             List<string> servers = null;
             if (string.IsNullOrEmpty(m_server))
             {
-                var dnslite = new DnsLib.DnsLite();
-                var dnslist = new List<string>();
+                var dnsclient = new LookupClient();
+                var records = dnsclient.Query(toMailDomain, QueryType.MX).Answers.MxRecords();

```

Comment:
For some reason, I'm not having any problems with the old `DnsLib.DnsLite` on the `experimental-net5-split` branch on Linux.  I'm not against using `DnsClient` from NuGet especially if it cleans up the code a bit, but would you be able to clarify the issues you were having with `DnsLite`?

Reply:
Based on my commit message (its been a while) the compiler was emitting a warning that certain functions dnslite used were not implemented on Linux/Mac.

Reply:
Thanks @tsuckow.  I see it the compiler warning now:
```
duplicati/thirdparty/DnsLite/DnsLite.cs(76,30): warning PC001: UdpClient.Receive(ref IPEndPoint) isn't supported on Linux and macOS
```